src/main/java/net/sf/mzmine/modules/visualization/tic/TICPlot.java

```diff
@@ -681,7 +681,7 @@ public String generateLabel(final XYDataset xyDataSet,
 		}
 	    });
 
-	    numOfPeaks++;
+	   // numOfPeaks++;
```

Comment:
I am not a big fan of such commented-out one-liners. Soon nobody is going to remember why this line is commented out. If the code is not needed, please remove it completely. If there is a reason to keep it, the reason should be explained in a comment.